docroot/themes/contrib/civic/includes/form.inc

```diff
@@ -324,7 +327,7 @@ function _civic_preprocess_composite_form_elements(&$variables) {
     $variables['type'] = $element['#type'];
     $variables['filter_text'] = $element['#title'];
     $variables['options_title'] = $element['#description'] ?? '';
-    $variables['is_inline'] = TRUE;
+    $variables['is_inline'] = FALSE;
```

Comment:
Desktop filters should be default as mobile can rely on JS since it is off-canvas and not visible on page load.

Reply:
And we can get  a flash on layout changing which is not what we want as we are using ajax for these pages.